azure-pipelines.yml

```diff
@@ -199,10 +199,20 @@ steps:
       package: '$(Build.ArtifactStagingDirectory)\teams-test-app\$(Build.BuildId).zip'
     displayName: 'Deploy to cloudroll'
 
+  - script: 'node prepNextDevRelease.js'
+    workingDirectory: '$(System.DefaultWorkingDirectory)\packages\teams-js'
+    condition: and(
+      eq(variables['Build.SourceBranch'], 'refs/heads/2.0-preview'),
+      in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'),
+      ne(variables['Build.Reason'], 'PullRequest'),
+      ne(variables['Build.Reason'], 'Manual')
+      )
+    displayName: 'node prepNextDevRelease.js'
```

Comment:
I just realized this step needs to happen before the build step on line 46 (and actually should happen before the ESLint task on line 38 too).

Reply:
I'll go ahead and move it, but to learn more about it- Is that because any edits made need to be linted and built out of principle? Results-wise, with the tasks that are between lines 38 - 210, it seems it'll be the same.